datastore/usermanager.go

```diff
@@ -86,20 +86,22 @@ const alterUserAPIKeyNotNullable = `alter table userinfo
 //
 // * Invalid:	default value set in case there is no authentication previous process for this user and thus not got a valid role.
 // * Standard:	role for regular users. This is the less privileged role
+// * SyncUser:	role for users that will used the Sync API
 // * Admin:		role for admin users, including standard role permissions but not superuser ones
 // * Superuser:	role for users having all the permissions
 const (
 	Invalid   = iota       // 0
 	Standard  = 100 * iota // 100
 	Admin                  // 200
 	Superuser              // 300
+	SyncUser  = 150
```

Comment:
mm.. what about having direct assignments instead of using iota?. Looks weird now having a mix of iota, value or nothing.

Another option, I've seen in lxd project that they redefine the int type for situations like this:

```
type operationClass int

const (
	operationClassTask      operationClass = 1
	operationClassWebsocket operationClass = 2
	operationClassToken     operationClass = 3
)

func (t operationClass) String() string {
	return map[operationClass]string{
		operationClassTask:      "task",
		operationClassWebsocket: "websocket",
		operationClassToken:     "token",
	}[t]
}
```